src/helpers/shared/filters.js

```diff
@@ -0,0 +1,49 @@
+export const getFiltersFromUrl = (keys, defaultValues = {}) => {
+  const searchParams = new URLSearchParams(location.search);
+
+  let filters = {};
+
+  keys.forEach((key) => {
+    const values = searchParams.getAll(key);
+    filters = {
+      ...filters,
+      [key]: values.length > 1 ? values : values[0],
```

Comment:
Is it 100% certain that the `values[0]` entry will exist? `searchParams.getAll` can return an empty array.

Reply:
No, it does not have to exist. But When the `getAll` function returns an empty array, then `values[0]` returns `undefined` which is not a problem in this case.